python/lsst/synpipe/compareModel.py

```diff
@@ -103,8 +100,7 @@ def main(root, visit, ccds, galType='sersic', output='outputs/'):
 
     npTable = writeNumpyTable(fakeTable)
     rerunName = root.split('/')[-2]
-    fitsTable = astropy.table.Table(npTable).write(output+'/'+rerunName +
-                                                   '_galMags.txt', format='ascii')
+    astropy.table.Table(npTable).write(output+'/'+rerunName + '_galMags.txt', format='ascii')
```

Comment:
Add some extra spaces around the  + while you're already editing this line. (Looks like it's not  approaching 110)